.github/workflows/build.yml

```diff
@@ -2,7 +2,28 @@ name: 🛠️ Build + Test
 
 on:
   push:
+    paths:
+      - '*.{js,ts,cjs,mjs,d.cts,d.mts,jsx,tsx,json,jsonc}'
```

Comment:
- '*.{js,ts,cjs,mjs,d.cts,d.mts,jsx,tsx,json,jsonc}'
      - 'src/core/icons/{file,folder,language}Icons.ts'

Isn't the 2nd line already covered by the first? I mean, we're running this pipeline once a "*.ts" file is changed, right? So why do we have to explicitly mention the {file,folder,language}Icons.ts path?